app/views/lead_providers/school_details/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render ::SchoolInformation::Details::BreadcrumbComponent.new(selected_cohort: @selected_cohort) %>
+
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+  <%= render ::SchoolInformation::Details::HeadComponent.new(school: @school, selected_cohort: @selected_cohort) %>
+  <%= render ::SchoolInformation::Details::MainInfoComponent.new(school: @school) %>
+</main>
```

Comment:
Why do we need a `main` tag here? I expect it would come from `application.html`.

Reply:
Good point

Reply:
Done